res/doc

```diff
@@ -24,7 +24,7 @@
 #                   selection - the selection menu (4.20.14622+)
 #                               all arguments will support substitutions in the form {A|B|C} as follows (see the end of this file for examples):
 #                                   A is string to substitute
```

Comment:
I was actually talking about the doc itself, ie., those A/B/C  ^^.

Reply:
Hmm. I'm not sure what I want to do with those. If I replace it with a full word rather than a letter, it may give the wrong impression.

Reply:
The main issue I have is simply that the section names matches a potential token (A & C). So, better sections names or better token names, the choice is yours ;).

Reply:
I'm not sure about the token names either... I originally used ones like `l`, `L`, `i`, and so on, but it ended up becoming nearly impossible to distinguish them except with a few fonts, so I switched to sequential letters instead. Do you have any specific suggestions?

Reply:
The ZSH docs I linked to earlier have some overlap with this, but I wouldn't necessarily use it as gospel either ;).

Random ideas:

t/s (for text/string/selection) instead of 1.

l for lowercase
U for uppercase
W for !word (inspired by Python re)
t for trim
C for for trim+collapse
e for empty/error
T for full trim
_ for underscore

Reply:
Done.